### ISSUE TITLE
Add support for Azure

### ISSUE THREAD

**Issue by yellow1912** [1up-lab/OneupFlysystemBundle#67]
I'm interested in adding support for https://github.com/thephpleague/flysystem-azure

Please let me know if you will be interested in pulling a pull request for that.

**Comment by bytehead** [comment]
You're always welcome to provide pull requests :)

**Comment by bytehead** [comment]
See https://github.com/1up-lab/OneupFlysystemBundle/pull/65#issuecomment-121186847

**Comment by alexandrubau** [comment]
+1

**Comment by willemstuursma** [comment]
I'm also interested in getting support for Azure, at the moment is is not possible to use this bundle to configure Azure blob storage for Symfony use.

**Comment by frco9** [comment]
The `league/flysystem-azure` package is now deprecated this one should now be used instead: `league/flysystem-azure-blob-storage`

And the `service.yaml` should be updated like so: 
```yaml
  flysystem.azureadapter:
    class: League\Flysystem\AzureBlobStorage\AzureBlobStorageAdapter
    arguments:
      - '@microsoftazure.storage.blobservice'
      - '%env(resolve:AZURE_STORAGE_CONTAINER_NAME)%'

  microsoftazure.storage.blobservice:
      class: 'MicrosoftAzure\Storage\Blob\BlobRestProxy'
      factory: ['MicrosoftAzure\Storage\Blob\BlobRestProxy','createBlobService']
      arguments:
        - 'DefaultEndpointsProtocol=https;AccountName=%env(resolve:AZURE_STORAGE_ACCOUNT_NAME)%;AccountKey=%env(resolve:AZURE_STORAGE_ACCOUNT_KEY)%;EndpointSuffix=core.windows.net'

```

**Comment by bytehead** [comment]
Implemented via #197.
